vllm/distributed/parallel_state.py

```diff
@@ -684,6 +861,28 @@ def get_tensor_model_parallel_rank():
     return get_tp_group().rank_in_group
 
 
+def get_pipeline_model_parallel_world_size():
+    """Return world size for the pipeline model parallel group."""
+    return get_pp_group().world_size
+
+
+def get_pipeline_model_parallel_rank():
+    """Return my rank for the pipeline model parallel group."""
+    return get_pp_group().rank_in_group
+
```

Comment:
these two are not used

Reply:
Do you mean they are not used currently? Planning to use them in next PRs

Reply:
yes, they are not used. in the future i will remove legacy usage like `get_tensor_model_parallel_rank`, as pointed in https://github.com/vllm-project/vllm/pull/5293#discussion_r1636941661 . The basic idea is that, "users" of `parallel_state` can assemble the functionality they want, rather than keep adding new helper functions in `parallel_state` .

Reply:
Understood, basically everything is done through `GroupCoordinator`. Removed all the extra helper functions.